index.bs

```diff
@@ -93,12 +118,24 @@ Introduction {#intro}
 ============
 
 Today, user agents generally identify themselves to servers by sending a `User-Agent` HTTP request
-header field along with each request (defined in Section 5.5.3 of [[RFC7231]]). Ideally, this header
-would give servers the ability to perform content negotiation, sending down exactly those bits that
-best represent the requested resource in a given user agent, optimizing both bandwidth and user
-experience. In practice, however, this header's value exposes far more information about the user's
-device than seems appropriate as a default, on the one hand, and intentionally obscures the true
-user agent in order to bypass misguided server-side heuristics, on the other.
+header field along with each request (defined in Section 5.5.3 of [[RFC7231]]). This header value
+gives servers the option to perform content negotiation, sending down exactly those bits that best
+represent the requested resource in a given user agent, optimizing both bandwidth and user
+experience. In addition to performance improvements the information also supports analytics, fraud
+detection, problem diagnosis and market share analysis.
+
+When combined with other information the `User-Agent` field value could be used to track specific 
+user agents and therefore might represent a risk to people's privacy. This problem is most severe 
+when content is served over insecure HTTP where access providers can view the information. Privacy
+concerns are mitigated to some extent when secure protocols such as HTTPS are used. Only parties 
```

Comment:
Moving to HTTPS is not sufficient to remove concerns related to fingerprinting, so this part should just be removed I think.